src/promptflow-tools/promptflow/tools/yamls/embedding.yaml

```diff
@@ -30,6 +30,10 @@ promptflow.tools.embedding.embedding:
       - text-search-ada-doc-001
       - text-search-ada-query-001
       allow_manual_entry: true
+    dimensions:
```

Comment:
maybe we should make this as 'advanced' parameter ?

Reply:
Yes, it might be better. 
I will update the code later this week, busy with graduation project and work these days.